src/test/groovy/org/jetbrains/intellij/IntelliJPluginSpecBase.groovy

```diff
@@ -44,6 +44,12 @@ abstract class IntelliJPluginSpecBase extends Specification {
                 downloadSources = false
                 intellijRepo = '$intellijRepo'
             }
+
+            // Define tasks with a minmal set of tasks required to build a source set
+            sourceSets.all {
```

Comment:
In the previous version of the tests you've always executed only the minimal set of tasks that was necessary to do the tests. There is no task that builds a source set so I've added one. We could also execute `jar` or `assemble` but that would also build the JAR file and potentially run some other tasks. I can change it if you want.